src/main/java/com/myplantdiary/enterprise/dao/SpecimenDAOStub.java

```diff
@@ -15,8 +15,7 @@
 
     @Override
     public Specimen save(Specimen specimen) throws Exception {
-        Integer specimenID = Integer.parseInt(specimen.getSpecimenId());
-        allSpecimens.put(specimenID, specimen);
+        allSpecimens.put(specimen.getSpecimenId(), specimen);
```

Comment:
I changed the specimen ID from string to int, so that it will be consistent, and will save duplicated lines of code.
Source: https://docs.sonarqube.org/display/SONARQUBE45/Duplications